test/object-store/util/sync/sync_test_utils.cpp

```diff
@@ -239,15 +243,44 @@ AutoVerifiedEmailCredentials create_user_and_log_in(app::SharedApp app)
 {
     REQUIRE(app);
     AutoVerifiedEmailCredentials creds;
-    app->provider_client<app::App::UsernamePasswordProviderClient>().register_email(
-        creds.email, creds.password, [&](util::Optional<app::AppError> error) {
-            REQUIRE(!error);
-        });
-    app->log_in_with_credentials(realm::app::AppCredentials::username_password(creds.email, creds.password),
-                                 [&](std::shared_ptr<realm::SyncUser> user, util::Optional<app::AppError> error) {
-                                     REQUIRE(user);
-                                     REQUIRE(!error);
-                                 });
+    int retry_count = 0;
```

Comment:
Is this to make sure the users can log in (given the network faults) so a test can start? I wonder if we can somehow whitelist these requests

Reply:
This was some of my testing - I'm actually going to revert these changes and approach this differently. e.g., use the direct baas admin port for all HTTP (non-websocket) operations.